ProcessMaker/Http/Resources/V1_1/CaseResource.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace ProcessMaker\Http\Resources\V1_1;
+
+use ProcessMaker\Http\Resources\ApiResource;
+
+class CaseResource extends ApiResource
+{
+    protected static $defaultFields = [
```

Comment:
Add documentation to clarify this are the field that are going to be returned as response of the endpoint